Frameworks/Foundation/NSIndexPath.mm

```diff
@@ -38,22 +49,28 @@ + (NSIndexPath*)indexPathWithIndexes:(const NSUInteger*)indexes length:(NSUInteg
  @Status Interoperable
 */
 + (NSIndexPath*)indexPathForRow:(NSInteger)row inSection:(NSInteger)section {
-    unsigned indexes[] = { section, row };
-
+    NSUInteger indexes[] = { section, row };
     return [[[self alloc] initWithIndexes:indexes length:2] autorelease];
 }
 
 /**
  @Status Interoperable
 */
-+ (NSIndexPath*)indexPathForItem:(NSInteger)row inSection:(NSInteger)section {
-    NSUInteger indexes[] = { section, row };
++ (NSIndexPath*)indexPathForItem:(NSInteger)item inSection:(NSInteger)section {
+    NSUInteger indexes[] = { section, item };
     return [[[self alloc] initWithIndexes:indexes length:2] autorelease];
 }
 
 /**
  @Status Interoperable
 */
+- (instancetype)init {
```

Comment:
According to documentation initWithIndexes:length: is also the designated initializer 🤣 (https://developer.apple.com/reference/foundation/nsindexpath/1416906-initwithindexes?language=objc) The previous implementation used initWithIndexes:length: as default so I'm going to leave it as is